cinema/serializers.py

```diff
@@ -1 +1,66 @@
-# write serializers here
+from rest_framework import serializers
+
+from cinema.models import Actor, CinemaHall, Genre, Movie, MovieSession
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ["id", "name"]
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = ["id", "first_name", "last_name", "full_name"]
```

Comment:
The `full_name` field in `ActorSerializer` needs to be explicitly defined using a `SerializerMethodField` or similar approach to ensure it is properly serialized. Consider adding a method `get_full_name` to return the `full_name` property.